### PR TITLE
Add notes about API Client

### PR DESCRIPTION
**Pull request by Shadowbeetle**
This Pull Request aims to better emphasize the need to use an API Client when integrating with API adapters.

### CODE REVIEW

File: lib/swoosh/adapters/amazon_ses.ex [hunk 1]
```diff
@@ -17,6 +17,10 @@ defmodule Swoosh.Adapters.AmazonSES do
         ]
       end
 
+  **All API adapters require an API client.** Swoosh currently supports Hackney and Finch.
```
Comment (princemaple) [inline lib/swoosh/adapters/amazon_ses.ex:20]:
```suggestion
**This adapter requires an API Client.** Swoosh comes with Hackney and Finch out of the box.
```

**Review by princemaple — CHANGES_REQUESTED** [review]

**Comment by Shadowbeetle** [comment]
Updated the docs in all files that used `Swoosh.ApiClient`. I updated my first commit based on your suggestion,  so it's strange why the `Changes requested` badge is still on.

**Comment by princemaple** [comment]
> Updated the docs in all files that used `Swoosh.ApiClient`. I updated my first commit based on your suggestion, so it's strange why the `Changes requested` badge is still on.

You have to click the re-request review button to get rid of it. It's next to the reviewer (me in this case) avatar in the reviewers list.

----

Thanks for the great effort! If there isn't more to be done in this PR, I'm happy to merge it in now.

**Review by princemaple — APPROVED** [review]

**Comment by Shadowbeetle** [comment]
> You have to click the re-request review button to get rid of it. It's next to the reviewer (me in this case) avatar in the reviewers list.

Thanks! Didn't know that.

> Thanks for the great effort! If there isn't more to be done in this PR, I'm happy to merge it in now.

I think it's ready to be merged. I'm happy I could help a bit!

**Comment by princemaple** [comment]
💜

**Pull request merged**